fluxcd/README.md

```diff
@@ -0,0 +1,135 @@
+
+#### Docker
+
+##### Metric collection
+
+This is an example configuration of a Docker label inside `docker-compose.yml`. See the [sample configuration file][4] for all available configuration options.
+
+```yaml
+labels:
+  com.datadoghq.ad.checks: '{"fluxcd":{"instances":[{"openmetrics_endpoint":"http://%%host%%:8080"}]}}'
+```
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Kubernetes" xxx -->
+
+#### Kubernetes
+
+##### Metric collection
+
+This is an example configuration with Kubernetes annotations on your Fluxcd pods. See the [sample configuration file][4] for all available configuration options.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/fluxcd.checks: |-
+      {
+        "fluxcd": {
+          "instances": [
+            {
+              "openmetrics_endpoint": "http://%%host%%:8080",
+            }
+          ]
+        }
+      }
+    # (...)
+spec:
+  containers:
+    - name: 'fluxcd'
+# (...)
+```
+
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
+
+Fluxcd metrics are available on the OpenMetrics endpoint. Additionally, Fluxcd allows you to [export custom application-level metrics][10]. You can configure the Fluxcd integration to collect these metrics using the `extra_metrics` option. All Fluxcd metrics, including your custom metrics, use the `fluxcd.` prefix.
+
+**Note:** Custom Fluxcd metrics are considered standard metrics in Datadog.
+
+This is an example configuration leveraging the `extra_metrics` option:
+
+```yaml
+init_config:
+  ...
+instances:
+  - openmetrics_endpoint: http://<FLUXCD_ADDRESS>:8080
+    # Also collect your own Fluxcd metrics
+    extra_metrics:
+      - my_custom_fluxcd_metric
+```
+
+See the [sample configuration file][4] for all available configuration options
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `fluxcd` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this integration.
+
+### Events
+
+The fluxcd integration does not include any events.
+
+### Service Checks
+
+See [service_checks.json][8] for a list of service checks provided by this integration.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][9].
+
+
+[1]: https://fluxcd.io/
+[2]: https://app.datadoghq.com/account/settings/agent/latest
+[3]: https://docs.datadoghq.com/agent/kubernetes/integrations/
+[4]: https://github.com/DataDog/integrations-core/blob/master/fluxcd/datadog_checks/fluxcd/data/conf.yaml.example
+[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[6]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
+[7]: https://github.com/DataDog/integrations-core/blob/master/fluxcd/metadata.csv
+[8]: https://github.com/DataDog/integrations-core/blob/master/fluxcd/assets/service_checks.json
+[9]: https://docs.datadoghq.com/help/
+[10]: https://docs.datadoghq.com/developers/integrations/python/
```

Comment:
Wrong link?

Reply:
Indeed! Actually that whole section where it's referenced doesn't really make sense for flux.